lib/rubocop/cop/style/unneeded_percent_q.rb

```diff
@@ -4,6 +4,16 @@ module RuboCop
   module Cop
     module Style
       # This cop checks for usage of the %q/%Q syntax when '' or "" would do.
+      # # bad
+      # name = %q(Bruce Wayne)
+      # time = %q(8 o'clock)
+      # question = %q("What did you say?")
+
+      # # good
+      # name = 'Bruce Wayne'
+      # time = "8 o'clock"
+      # question = '"What did you say?"'
+
```

Comment:
Please insert `#` in the following positions and then run `rake generate_cops_documentation`.

```diff
 # This cop checks for usage of the %q/%Q syntax when '' or "" would do.
+#
 # # bad
 # name = %q(Bruce Wayne)
 # time = %q(8 o'clock)
 # question = %q("What did you say?")
+#
 # # good
 # name = 'Bruce Wayne'
 # time = "8 o'clock"
 # question = '"What did you say?"'
+#
 class UnneededPercentQ < Cop
```